lib/utils/settings.html

```diff
@@ -16,21 +16,21 @@
 (function() {
   'use strict';
 
-  const settings = Polymer.Settings || {};
-  settings.useShadow = !(window.ShadyDOM);
-  settings.useNativeCSSProperties =
-    Boolean(!window.ShadyCSS || window.ShadyCSS.nativeCss);
-  settings.useNativeCustomElements =
-    !(window.customElements.polyfillWrapFlushCallback);
-
   /**
    * Sets the global, legacy settings.
    *
    * @deprecated
    * @namespace
    * @memberof Polymer
    */
-  Polymer.Settings = settings;
+   Polymer.Settings = Polymer.Settings || {};
+
+  Polymer.Settings.useShadow = !(window.ShadyDOM);
```

Comment:
Indentation is a bit off.

Reply:
bah, I'm too used to clang. Thanks for catching
Update: Fixed